### ISSUE TITLE
Make email and phone links clickable

### ISSUE THREAD

**Issue by codebryo** [caniscodo/vet-website#8]
Looked at the [Team Page](https://github.com/caniscodo/vet-website/blob/master/grabenwoeger-website/pages/team.vue) and I would make the links clickable so it's a nice experience for people finding the page on the phone.

e.g. `<a href="tel:+43699699699">0699 699 699</a>` or `<a href="mailto:herr.bauer@mail.com">herr.bauer@mail.com</a>`

**Comment by caniscodo** [comment]
Yeah, that makes sense, it seems I completely overlooked that. Thank you for the feedback!

Best regards,
Hannah
________________________________
From: Roman Kuba ***@***.***>
Sent: Wednesday, February 22, 2023 9:00:02 PM
To: caniscodo/vet-website ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [caniscodo/vet-website] Make email and phone links clickable (Issue #8)


Looked at the Team Page<https://github.com/caniscodo/vet-website/blob/master/grabenwoeger-website/pages/team.vue> and I would make the links clickable so it's a nice experience for people finding the page on the phone.

e.g. <a href="tel:+43699699699">0699 699 699</a> or <a ***@***.******@***.***</a>
